### PR TITLE
HugePages graduated to GA

### PR DESCRIPTION
**Pull request by derekwaynecarr**
HugePages have graduated to GA in 1.14.

### CODE REVIEW

File: content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ content_template: templates/task
 ---
 
 {{% capture overview %}}
-{{< feature-state state="beta" >}}
+{{< feature-state state="GA" >}}
```
Comment (sftim) [inline content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md]:
```suggestion
{{< feature-state state="stable" >}}
```
The [Hugo shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) for this uses the keyword `stable`

**Comment by DanyC97** [comment]
@derekwaynecarr are you planning to fix the comment @sftim made ? w/o it the Travis CI fails, thanks

**Pull request merged**